Samples/Hast.Samples.Consumer/Gui.cs

```diff
@@ -0,0 +1,356 @@
+using Hast.Layer;
+using Hast.Samples.Consumer.Models;
+using Microsoft.Extensions.Logging;
+using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Terminal.Gui;
+
+namespace Hast.Samples.Consumer
+{
+    public class Gui

```

Comment:
Lehet, hogy el kell találni pontosan a betűket, különben nem regisztrálja a kattintást? Mert nekem végig szenvedés volt kattintgatni, olyan, mintha néha regisztrálná, néha nem.

https://user-images.githubusercontent.com/1976647/139743366-5ce35280-ab00-4366-b0b9-26015898fbe2.mp4

Reply:
Érdekes, nálam nem volt ilyen.

Reply:
Most direkt próbálgattam és a soron belül tök mellé kattintva is felvette.
Viszont olyat tapasztaltam, hogy az első kattintást nem érzékelte. Ennek az az oka, hogy amikor a klikk történik a cmd ablak még nincs fókuszban. Azzal nem hiszem lehet mit kezdeni.

Reply:
A videó alapján úgy látom akkor van gond, amikor kattintásra bemozog az egér és így mouse up-ra már nem ott van a kurzor ahol mouse down-nál volt? Ha erről van szó, akkor az eltérő egér érzékenység miatt lehet más a felhasználói élményünk. Ezzel viszont az a gond, hogy a menü és lista beépített vezérlők és a kattintás eseményt a library kezeli le.

Reply:
Passz. Most a Device Name, Endpoint, Sample To Run-nál látható a szenvedésem. Nem tudom mi lehet a sajátosság abban, ahogy én kattintok, de máshol nincs ilyen probléma (sajnos a Screenpresso valamiért csak itt nem mutatja a kattintásokat).

https://user-images.githubusercontent.com/1976647/140627308-1dd3c72d-62f4-4c43-9ab2-e9b71cd52683.mp4

Reply:
Most jobb-e? Kikapcsoltam az `Application.UseSystemConsole`-t, ami azt jelenti, hogy a System.Console helyett natív hívásokat használ (Windows API, illetve nCurses).

Reply:
Nem tudom ezt kvantifikálni, de talán jobb. Most elég sokat kattintgattam (65940965f0a663035bf3479dab0540b1c4112378), és jobbnak tűnik, bár továbbra is előfordul ugyanez. De ezzel már együtt tudok élni.